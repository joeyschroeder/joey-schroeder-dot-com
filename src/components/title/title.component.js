import React from 'react';
import styles from './title.style.scss';

export const Title = () => (
  <h2 className={styles.root}>
    <span className={styles.word}>Designer//</span>
    <span className={styles.word}>Developer</span>
  </h2>
);
