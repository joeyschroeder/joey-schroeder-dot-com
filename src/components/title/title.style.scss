@import '../../styles/config/variables';
@import '../../styles/mixins/font-size';
@import '../../styles/mixins/line-height';

.root {
  @include font-size(4.8);
  @include line-height(4.8);

  margin: 0;
  text-transform: uppercase;

  .word {
    display: inline-block;
  }
}
