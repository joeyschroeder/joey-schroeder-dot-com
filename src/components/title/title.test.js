import React from 'react';
import { create } from 'react-test-renderer';
import { Title } from './title.component';

describe('<Title />', () => {
  it('should render correctly', () => {
    const component = create(<Title />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
