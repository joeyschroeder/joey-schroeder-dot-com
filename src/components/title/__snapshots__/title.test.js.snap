// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render correctly 1`] = `
<h2
  className="root"
>
  <span
    className="content"
  >
    Developer//
  </span>
  <span
    className="content"
  >
    Designer
  </span>
</h2>
`;
